src/utils/log.ts

```diff
@@ -21,8 +21,12 @@ class ConsoleLogger {
         console.info(chalk.red(`[MINYAMI][ERROR] ${message}`));
     }
 
-    enableDebugMode() {
-        this.isDebugMode = true;
+    setDebugMode(enabled: boolean): void {
+        this.isDebugMode = enabled;
+    }
```

Comment:
```suggestion
    setDebugMode(enabled: boolean) {
        this.isDebugMode = enabled;
    }
```

The codebase you are working on did not seem to specify a `void` returned value.

Reply:
Thank you for pointing that out. Please check my other comments above.

Reply:
In the future you can apply the suggestion by one click: [Incorporating feedback in your pull request - GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).